code/modules/antagonists/eldritch_cult/eldritch_magic.dm

```diff
@@ -667,3 +671,45 @@
 		return 3
 	else
 		return 2
+
+/obj/effect/proc_holder/spell/targeted/shed_human_form
+	name = "Shed form"
+	desc = "Shed your fragile form, become one with the arms, become one with the emperor."
+	invocation_type = INVOCATION_SHOUT
+	invocation = "REALITY UNCOIL!"
+	clothes_req = FALSE
+	action_background_icon_state = "bg_ecult"
+	range = -1
+	include_user = TRUE
+	charge_max = 1
+	action_icon = 'icons/mob/actions/actions_ecult.dmi'
+	action_icon_state = "worm_ascend"
+	var/segment_length = 10
+
+/obj/effect/proc_holder/spell/targeted/shed_human_form/cast(list/targets, mob/user)
+	. = ..()
+	var/mob/living/target = user
+	var/mob/living/mob_inside = locate() in target.contents - target
+
+	if(!mob_inside)
+		var/mob/living/simple_animal/hostile/eldritch/armsy/prime/outside = new(user.loc,TRUE,segment_length)
+		target.mind.transfer_to(outside, TRUE)
+		target.forceMove(outside)
+		for(var/mob/living/carbon/human/humie in view(9,outside))
+			if(IS_HERETIC(humie) || IS_HERETIC_MONSTER(humie))
+				continue
+			SEND_SIGNAL(humie, COMSIG_ADD_MOOD_EVENT, "gates_of_mansus", /datum/mood_event/gates_of_mansus)
+			///They see the very reality uncoil before their eyes.
+			if(prob(25))
+				var/trauma = pick(subtypesof(BRAIN_TRAUMA_MILD) + subtypesof(BRAIN_TRAUMA_MILD))
+				humie.gain_trauma(new trauma(), TRAUMA_RESILIENCE_LOBOTOMY)
+		return
+
+	if(ishuman(mob_inside))
+		var/mob/living/simple_animal/hostile/eldritch/armsy/prime/armsy = target
+		armsy.mind.transfer_to(mob_inside, TRUE)
+		mob_inside.forceMove(armsy.loc)
+		segment_length = armsy.get_length()
+		mob_inside.fully_heal(FALSE)
```

Comment:
uhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh

this spell has a cooldown of 1 decisecond, right? so can't you just murderbone people as a normal human and cast this twice to heal and unstun yourself whenever you want to? it's like that dextrous holoparasite+staff of healing wizard strategy, but even stronger.

Reply:
oh shit i forgot, i made this for debug purposes